src/ui.tsx

```diff
@@ -276,30 +290,6 @@ class UserInterface extends Component<Props, State> {
     imageFileInfo: ImageFileInfo,
     images: ImageBitmap[][]
   ) => {
-    const today = new Date();
-    const newMetadata = {
-      imageName: imageFileInfo.fileName,
-      id: imageFileInfo.fileID,
-      dateCreated: today.toLocaleDateString("gb-EN"),
-      size: imageFileInfo.size.toString(),
-      dimensions: `${imageFileInfo.width} x ${imageFileInfo.height}`,
-      numberOfDimensions: images.length === 1 ? "2" : "3",
-      numberOfChannels: images[0].length.toString(),
-      imageLabels: [] as Array<string>,
```

Comment:
We need some code that replaces this, to upload an image in CURATE when CURATE is not wrapped DOMINATE.

Reply:
Do we need CURATE to work as standalone? If so then we'll need some way for it to know whether DOMINATE is there or not.

Reply:
Yes, i think we want it to be standalone, similarly to what we do in ANNOTATE. All CURATE props are optional; some of these are specific to DOMINATE (for example, `saveImageCallback`, so if these props are set, then we know DOMINATE is there).